tests/test_browser_model.py

```diff
@@ -3,7 +3,7 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 
-from foxpuppet import FoxPuppet
+from foxpuppet.foxpuppet import FoxPuppet
```

Comment:
We should keep this as `from foxpuppet import FoxPuppet` for the sake of our users sanity. :smile: